main/py-chardet/APKBUILD

```diff
@@ -2,12 +2,12 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 _pkgname=chardet
 pkgname=py-$_pkgname
-pkgver=2.3.0
-pkgrel=3
+pkgver=3.0.3
+pkgrel=0
 pkgdesc="The Universal Character Encoding Detector"
 url="https://github.com/chardet/chardet/"
 arch="noarch"
-license="LGPL-2.1"
+license="LGPL 2.1"
```

Comment:
No, it should be [nothing](https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license). Seems I can't remember what I wrote 3.5 years back.